README.md

```diff
@@ -73,6 +73,10 @@ Several conditions can be added to the matcher:
 _The `*_event` condition can be called multiple times with different events._
 
 
+## Disabling
+
+RSpec Otel wraps every example in a new OpenTelemetry SDK configuration by default, if you wish to disable this you can tag your example with `:rspec_otel_disable_tracing`.
```

Comment:
```suggestion
We every example in a new OpenTelemetry SDK configuration by default, if you wish to disable this you can tag your example with `:rspec_otel_disable_tracing`:

```ruby
require 'spec_helper'
RSpec.describe 'User API', :rspec_otel_disable_tracing do
  it 'tests my code' do
    expect(true).to be true
  end
end
```

```

It seems adding a code sample breaks the update suggestion. But I suppose you see the point of adding an example :)